src/components/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Logo from '../images_and_logos/logo.svg';
+import '../Styling/Header.css';
+
+function Header() {
+  return (
+    <>
+      <header>
+        <Link to="/">
+          <img className="Logo" src={Logo} alt="Reddit Timer Logo" />
```

Comment:
Here you use global CSS classes. It's better to use CSS modules if you don't want to go for styled-components. 

It sounds more complicated than it actually is. The only thing you need to do is change the import of the css file to 

```
import styles from '../Styling/Header.css`;
```

and use the classes like this

```
<img className={styles.Logo} src={Logo} alt="Reddit Timer Logo" />
```

Now the class names are automatically scoped to the component, meaning that they get a random prefix. This way they won't conflict when you use duplicate class names inside the app.

It also makes sense to keep the style files next to the components and not in a separate `Styling` folder. I'd say that's best practice in the React world

Reply:
Oh wow I didn't even think to do that, I will keep that in mind on future projects and fix this one accordingly. I'm surprised that you wouldn't want all the style in a folder and all the pages or components in their own, though it does make sense to keep the files associated closer, I figured a folder may get too big if both the page and styles are kept in it together.